### PR TITLE
Compile raylet cython bindings with bazel

### PR DESCRIPTION
**Pull request by pcmoritz**
The plasma store can be built with:
```
bazel build @plasma//:plasma_server -c opt
```

And the raylet can be built with:
```
bazel build //:_raylet -c opt
```

Related to #2887.

### CODE REVIEW

File: bazel/cython_library.bzl [hunk 1]
```diff
@@ -0,0 +1,71 @@
+"""Custom rules for gRPC Python"""
```
Comment (robertnishihara) [inline bazel/cython_library.bzl]: What is the role of this file?

File: WORKSPACE [hunk 2]
```diff
@@ -22,8 +22,20 @@ git_repository(
 )
 
 new_git_repository(
-    name = "plasma",
-    build_file = "@//bazel:BUILD.plasma",
```
Comment (robertnishihara) [inline WORKSPACE]: original indentation was correct

File: bazel/python_configure.bzl [hunk 3]
```diff
@@ -0,0 +1,295 @@
+# Adapted with modifications from tensorflow/third_party/py/
```
Comment (robertnishihara) [inline bazel/python_configure.bzl]: Maybe add a comment pointing out the minor modifications you made to this file

**Review by robertnishihara — APPROVED** [review]

**Comment by robertnishihara** [comment]
@rsepassi please take a quick look as a sanity check if you have a chance!

**Comment by rsepassi** [comment]
looks good!

**Comment by robertnishihara** [comment]
Thanks @rsepassi!

**Pull request merged**